packages/airnode-adapter/src/response-processing/extraction.test.ts

```diff
@@ -66,11 +64,95 @@ describe('extractAndEncodeValue', () => {
   it('extracts and encodes the value from complex objects', () => {
     const data = { a: { b: [{ c: 1 }, { d: '750.51' }] } };
     const parameters: ReservedParameters = { _path: 'a.b.1.d', _type: 'int256', _times: '100' };
-    const res = extraction.extractAndEncodeResponse(data, parameters);
+    const res = extractAndEncodeResponse(data, parameters);
     expect(res).toEqual({
-      rawValue: '750.51',
-      value: '75051',
+      rawValue: data,
+      values: ['75051'],
       encodedValue: '0x000000000000000000000000000000000000000000000000000000000001252b',
     });
   });
+
+  it('accepts "" (empty string) for _times parameter', () => {
```

Comment:
Since we allow using empty string when specifying multiple parameters It makes sense to support this for single values as well (it's more consistent and easier to document).